newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -217,11 +238,17 @@ const BehaviorsEditor = (props: Props) => {
               );
               const iconUrl = behaviorMetadata.getIconFilename();
 
+              const ref =
```

Comment:
Note super pretty but does the job :)

Reply:
Yeah there is a similar logic inside the `ExpressionAutocompletionsDisplayer` which is where I copied it